services/db/workspaceTableHandler.go

```diff
@@ -0,0 +1,104 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/rudderlabs/rudder-server/config"
+)
+
+var (
+	host, user, password, dbname string
+	port                         int
+)
+
+type WorkspaceDataT struct {
+	token      string
+	created_at string
```

Comment:
We use camelCase for var names in go. Postgres columns remain snake case. Please change this to createdAt.